.jenkins/pytorch/macos-test.sh

```diff
@@ -53,7 +53,7 @@ test_python_all() {
 
   # Try to pull value from CIRCLE_PULL_REQUEST first then GITHUB_HEAD_REF second
   # CIRCLE_PULL_REQUEST comes from CircleCI
-  # GITHUB_HEAD_REF comes from Github Actions
+  # NOTE: file_diff_from_base is currently bugged for GHA due to an issue finding a merge base for ghstack PRs
```

Comment:
can these link to a GitHub issue for more context/tracking?

Reply:
Added a reference to https://github.com/pytorch/pytorch/issues/60111